kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -131,12 +135,18 @@ public GetBatchesResponse listBatches(
   }
 
   public OperationLog getBatchLocalLog(String batchId, int from, int size) {
+    return getBatchLocalLog(batchId, from, size, Collections.emptyMap());
+  }
+
+  public OperationLog getBatchLocalLog(
+      String batchId, int from, int size, Map<String, String> additionalHeaders) {
```

Comment:
prefer a short name if it's clear enough. additionalHeaders => headers

an additional thing needs to be clarified, what will happen if the user supplies reserved headers, which one will take effect?